oauth2client/contrib/django_util/models.py

```diff
@@ -0,0 +1,60 @@
+# Copyright 2016 Google Inc.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import base64
+import pickle
+
+from django.db import models
+from django.utils.encoding import smart_bytes, smart_text
+
+import oauth2client
+
+
+class CredentialsField(models.Field):
+    def __init__(self, *args, **kwargs):
+        if 'null' not in kwargs:
+            kwargs['null'] = True
+        super(CredentialsField, self).__init__(*args, **kwargs)
+
+    def get_internal_type(self):
+        return 'TextField'
```

Comment:
changed to BinaryField. TextField is definitely wrong ,CharField is a contender but I agree this is byte-y enough that BinaryField feels most appropriate.